llvm/utils/repo/Makefile

```diff
@@ -8,7 +8,12 @@
 
 # The path of the directory in which llvm-project-prepo sources is checked out from git.
 LLVM_REPO = ${HOME}/llvm-project-prepo
-TOOLCHAIN_FILE=$(LLVM_REPO)/llvm/utils/repo/repo.cmake
+# Choose the toolchina file to be used. By default, the repo.cmake is used.

```

Comment:
s/toolchina/toolchain.

> By default, the repo.cmake is used.

The extra “the” is superfluous here: “By default, repo.cmake is used"

Reply:
Thanks, fixed it.